src/Commands/ArtCommand.php

```diff
@@ -88,8 +87,46 @@ protected function availableArt()
                 function ($file) {
                     return $file->getBasename('.txt');
                 },
-                (array) $finder->getIterator()
+                (array)$finder->getIterator()
             )
         );
     }
+
+    /**
+     * Set the art filename.
+     *
+     * @param $name
+     *
+     * @return \Pantheon\Terminus\Commands\ArtCommand
+     *
+     */
+    public function setFilename($name)
```

Comment:
I'm fine with leaving the name as it is (since I'm not sure what else I'd name it), but make it protected.